pkg/bosh/manifest/cmd_render_job_tmpls.go

```diff
@@ -152,54 +152,55 @@ func runRenderScript(
 		// Run the pre-render script.
 		cmd := exec.Command("/bin/bash", tmpFile.Name())
 
-		if !silent {
-			errReader, err := cmd.StderrPipe()
-			if err != nil {
-				return createErr(err)
-			}
-			outReader, err := cmd.StdoutPipe()
-			if err != nil {
-				return createErr(err)
-			}
+		errReader, err := cmd.StderrPipe()
+		if err != nil {
+			return createErr(err)
+		}
+		outReader, err := cmd.StdoutPipe()
+		if err != nil {
+			return createErr(err)
+		}
 
-			errScanner := bufio.NewScanner(errReader)
-			go func() {
-				for errScanner.Scan() {
-					fmt.Printf("pre-render-err-%d | %s\n", idx, errScanner.Text())
-				}
-			}()
+		outBuffer := bytes.NewBufferString("")
```

Comment:
Using `strings.Builder` is preferable here. You can simply:
```go
var outBuffer, errBuffer strings.Builder
```
It will initialize the buffers correctly.